packages/core/src/components/tabs/test/inline-tabs/inline-tabs.e2e.ts

```diff
@@ -35,7 +35,6 @@ test.describe.parallel('tds-inline-tabs', () => {
 
   test('selected tab index should be 0', async ({ page }) => {
     await page.waitForChanges();
```

Comment:
Accidentally added this and the one below?

Reply:
It was due to merge conflict.